README.md

```diff
@@ -305,6 +363,10 @@ Built-in function ready to be bound to any key you like :smile:.
 | `builtin.reloader`                  | Lists lua modules and reload them on enter                       |
 | `builtin.vim_options`               | Lists vim options and on enter edit the options value            |
 | `builtin.keymaps`                   | Lists normal-mode mappings                                       |
+| `builtin.git_commits`               | Lists git commit with diff preview and checkout action           |
+| `builtin.git_bcommits`              ||
```

Comment:
Lists git commits for the current buffer

Reply:
Ohhhhhh I was struglling to understand what bcommit does, Thats an amazing feature, very useful thanks a lot